app/components/windows/WidgetEditor.vue

```diff
@@ -17,7 +17,7 @@
           </div>
         </div>
         <div v-if="apiSettings.testers" class="button button--action test-button">
-          <test-widgets :componentProps="{ testers: apiSettings.testers }" />
+          <test-widgets :componentProps="{ testers: null }" />
```

Comment:
i think we probably just want to remove this prop or make it optional rather than passing in null right?